plugins/scaffolder-backend-module-bitbucket/src/actions/bitbucket.examples.ts

```diff
@@ -100,17 +100,14 @@ export const examples: TemplateExample[] = [
           input: {
             repoUrl:
               'bitbucket.org?repo=repo&workspace=workspace&project=project',
-            description: 'Initialize a git repository',
-            repoVisibility: 'public',
-            defaultBranch: 'main',
```

Comment:
@tudi2d it's the removal of these lines from the examples which results in the default branch being used, which is master.